src/main/java/com/venafi/vcert/sdk/connectors/tpp/AbstractTppConnector.java

```diff
@@ -0,0 +1,180 @@
+package com.venafi.vcert.sdk.connectors.tpp;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.gson.annotations.SerializedName;
+import com.venafi.vcert.sdk.connectors.ServerPolicy;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public abstract class AbstractTppConnector {
+    protected static final Pattern POLICY_REGEX = Pattern.compile("^\\\\VED\\\\Policy");
+    protected static final String HEADER_VALUE_AUTHORIZATION = "Bearer %s";
+
+    protected static final String MISSING_CREDENTIALS_MESSAGE = "failed to authenticate: missing credentials";
+
+
+    protected final Tpp tpp;
+
+    @Getter
+    protected String zone;
+    @Getter
+    protected String vendorAndProductName;
+    protected static final String tppAttributeManagementType = "Management Type";
```

Comment:
this is constant so capitalize the name please

Reply:
Updated the name to match Java constants guideline.